cloudera-integration/csd/aux/kafka.properties

```diff
@@ -0,0 +1,9 @@
+bootstrap.servers=master1.valhalla.phdata.io:9093,master2.valhalla.phdata.io:9093,master3.valhalla.phdata.io:9093
```

Comment:
These are environmental configs so we shouldn't commit them into the project. Just leave this and the trust store properties out, then we need to document them.

Reply:
Should I leave `kafka.properties` as empty file?